earth_enterprise/rpms/opengee-server/pre-uninstall.sh

```diff
@@ -17,16 +17,12 @@
 set +x
 
 #-----------------------------------------------------------------
-# Main Functions
+# Main Function
 #-----------------------------------------------------------------
-main_preuninstall()
-{
-    service geserver stop
-    remove_service geserver
-}
 
-#-----------------------------------------------------------------
-# Pre-install Main
-#-----------------------------------------------------------------
+service geserver stop
```

Comment:
You don't need to stop the service if this is an upgrade. You should only be doing things that don't effect a running service at this point. Or if you're supper worried about that, you can stop it, do your work, and then start it again.

Reply:
You do have to stop it if it is an upgrade, because you are replacing the current executables with new ones. Well, you could do condrestart, but I wasnt sure if the current scripts supported that.